tools/packages/wesl/src/debug/ASTtoString.ts

```diff
@@ -179,7 +179,7 @@ function addFnFields(elem: FnElem, str: LineWrapper) {
   const paramStrs = params
     .map(
       (
-        p, // TODO DRY
+        p, // LATER DRY
```

Comment:
I suspect that ASTtoString will undergo some more rewrites in the future, so I think it's fine to defer this.